tests/forms_tests/tests/test_formsets.py

```diff
@@ -1748,9 +1750,9 @@ def test_management_form_invalid_data(self):
             "</ul>"
             "<div>"
             '<input type="hidden" name="form-TOTAL_FORMS" value="two" '
-            'id="id_form-TOTAL_FORMS">'
+            'aria-invalid="true" id="id_form-TOTAL_FORMS">'
```

Comment:
@django/accessibility do you think we should still be adding this to "hidden" fields. In this case it's the formset management forms? 

I guess as long as "hidden" fields are not read by screen readers it's ok 🤔

Reply:
According to https://www.w3.org/TR/html-aria/#input-hidden we shouldn't add any aria attributes to hidden inputs.